standard/variables.md

```diff
@@ -802,6 +802,63 @@ For a *lambda_expression* or *anonymous_method_expression* *expr* with a body (e
 >
 > *end example*
 
+#### §definite-assignment-rules-for-local-function Rules for variables in local functions
+
+Local functions are analyzed in the context of their parent method. There are two control flow paths that matter for local functions: function calls and delegate conversions.
+
+Definite assignment for the body of each local function is defined separately for each call site. At each invocation, variables captured by the local function are considered definitely assigned if they were definitely assigned at the point of call. A control flow path also exists to the local function body at this point and is considered reachable. After a call to the local function, captured variables that were definitely assigned at every control point leaving the function (`return` statements, `yield` statements, `await` statements) are considered definitely assigned after the call location.
```

Comment:
"captured" is not defined.

Reply:
It's defined for anonymous function expressions in https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/expressions.md#111662-captured-outer-variables.

We could potentially leave it here without definition, but raise an issue to extract the concept of a captured variable to a common place and reference it from 11.16.6.2 and here.

Reply:
See #622

No change at this time. This should be addressed as part of #622